server/main.go

```diff
@@ -31,6 +35,7 @@ func main() {
 	// Websocket setup (package webserver)
 	wb := webserver.NewWebBroker(webBroadcastCh, webResponseCh, &wgQuit)
 	go wb.RunLoop() // Run the web broker loop asynchronously
+	fmt.Printf("[Start] Server running on :%v\n", PORT)
```

Comment:
The port is not a constant, and is specified in the config.json file at the top level of the repo. I like your idea of exposing the server port on the command line, but please use conf.WebSocketPort to specify the port so that it updates with the configuration.